web-client/integration-tests/adcSignsProposedStipDecision.test.js

```diff
@@ -45,8 +44,8 @@ describe('Sr. Attorney Signs Proposed Stipulated Decision', () => {
     test,
     'Jeff, this is ready for review and signature',
   );
-  docketClerkVerifiesStipulatedDecisionExistsInInbox(
-    test,
-    'Donna, this is not ready to serve. I need to follow up on something first',
-  );
+  // docketClerkVerifiesStipulatedDecisionExistsInInbox(
```

Comment:
Should there be a note about why this is commented out?

Reply:
I'll delete it, it's no longer needed.

Reply:
Updated!